Library/Homebrew/test/cmd/list_spec.rb

```diff
@@ -1,12 +1,10 @@
 # frozen_string_literal: true
 
-require "cmd/shared_examples/args_parse"
+require "cmd/list"
 
-RSpec.describe "brew list" do
+RSpec.describe Homebrew::Cmd::List do
```

Comment:
Note that once we've ported all the commands, we can/should enable `RSpec/DescribeClass` (the commands are all currently in the `Homebrew` namespace, so the current approach makes sense).